src/geostats.jl

```diff
@@ -117,8 +117,8 @@ function solvesingle(::SimulationProblem, covars::NamedTuple, solver::IQ, prepro
     reals = iqsim(trainimg, par.tilesize, simsize;
                   overlap=overlap, path=par.path,
                   soft=par.soft, hard=hard, tol=par.tol,
-                  threads=solver.threads, gpu=solver.gpu,
-                  showprogress=solver.showprogress, rng=rng)
+                  threads=solver.threads,
+                  showprogress=solver.showprogress, rng=rng, resource=solver.resource)
```

Comment:
Please follow the order of arguments specified in the docstring. Things start to become a mess quickly if we are not consistent with these small details.

Reply:
Ok. Fixed.

Reply:
Yes. I have moved it to the end both in the IQ structure and in the docstring, in my local repository.